src/main/java/de/sakpaas/backend/v2/controller/UserController.java

```diff
@@ -29,6 +46,27 @@ public UserController(UserService userService) {
   @GetMapping("/info")
```

Comment:
Den Endpunkt sollten wir später noch in die openAPI aufnehmen. Aber finde ich eigentlich gut. :)

Reply:
Jo, auch die Credits gehen glaube ich an @RobSpot-Gaming :D